test/spec/ol/map.test.js

```diff
@@ -601,21 +600,6 @@ describe('ol.Map', function() {
         const interactions = defaultInteractions(options);
         expect(interactions.getLength()).to.eql(1);
         expect(interactions.item(0)).to.be.a(PinchZoom);
-        expect(interactions.item(0).constrainResolution_).to.eql(false);
-      });
-    });
-
-    describe('set constrainResolution option', function() {
```

Comment:
Have these tests also been moved elsewhere?

Reply:
Since the `constrainResolution` option does not exist on interactions anymore, I felt this test was rendered useless. I may be wrong though.